### PR TITLE
Ldap password renewal fixes for NC13

### PR DESCRIPTION
**Pull request by GitHubUser4234**
Hi,

The "Renew Password" page in user_ldap needed minor adjustments to keep it aligned with NC13 core changes:

- Commit _"adjust css to nc13 core changes"_
Since NC 13 the "Renew Password" page doesn't load the `server.scss` anymore, so tooltip related CSS is copied over to `renewPassword.css` until issue #9251 is solved. The commit also includes a minor adjustment in label position and button background color

- Commit _"remove reset password link"_
The reset password function in core has been heavily revamped, so instead of updating the "Renew Password" page with this now quite complicated function, the reset password link is now removed completely from the "Renew Password" page. This is because such feature isn't needed on the "Renew Password" page anyway, and here's why: 
1) User has to enter the correct password on the Login page before being redirected to the "Renew Password" page, so it's unlikely he suddenly forgets the password after having entered it correctly on the previous page
2) User could click "Cancel" on the "Renew Password" page to return the Login page and perform a password reset there

### CODE REVIEW

File: apps/user_ldap/css/renewPassword.css [hunk 1]
```diff
@@ -19,7 +19,109 @@
...
+ text-align:center !important;
+ font-weight:normal !important;
+ border-radius:3px
+}
+.tooltip-arrow {
+ position:absolute;
+ width:0;
+ height:0;
+ border-color:transparent;
+ border-style:solid
```
Comment (MorrisJobke) [inline apps/user_ldap/css/renewPassword.css]: Could I ask you to have them same indention level as in the previous lines.

Comment (GitHubUser4234) [reply apps/user_ldap/css/renewPassword.css]: Oh yeah sure, done now with commit "_css indent alignment_"

**Comment by MorrisJobke** [comment]
cc @nextcloud/ldap

**Comment by blizzz** [comment]
> The commit also includes a minor adjustment in label position and button background color

Is it specific to that page, or useful for nextcloud in general?

**Comment by GitHubUser4234** [comment]
> Is it specific to that page, or useful for nextcloud in general?

Specific to that page.

**Review by blizzz — APPROVED** [review]
Form looks proper, logic works

**Comment by blizzz** [comment]
needs another reviewer

**Review by rullzer — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Backport is in #9661

**Pull request merged**